tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -53,7 +53,9 @@ namespace Vkgc {
 std::ostream &operator<<(std::ostream &out, VkVertexInputRate inputRate);
 std::ostream &operator<<(std::ostream &out, VkFormat format);
 std::ostream &operator<<(std::ostream &out, VkPrimitiveTopology topology);
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION < 48
```

Comment:
Can we just remove these code without "#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION < 48"?
since you have ignore it in PatchResourceCollect.cpp without any version check.
looks good to me for others.

Reply:
This is to keep backward compatibility in case any one wants to dump it. Just a dummy info now.